src/test/java/org/assertj/core/internal/paths/Paths_assertIsAbsolute_Test.java

```diff
@@ -0,0 +1,65 @@
+package org.assertj.core.internal.paths;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.internal.PathsBaseTest_NoFileSystem;
+import org.junit.Test;
+
+import java.nio.file.Path;
+
+import static org.assertj.core.error.ShouldBeAbsolutePath.shouldBeAbsolutePath;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.test.TestFailures.failBecauseExpectedAssertionErrorWasNotThrown;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings("AutoBoxing")
+public class Paths_assertIsAbsolute_Test
+    extends PathsBaseTest_NoFileSystem
```

Comment:
The problem is that for some assertions I'll need more than one path (example: test that two paths share the same filesystem; test that the target of a symlink is a given path; etc etc).

For tests that only use paths, I really want to avoid having to initialize a FileSystem. What is more, and that is a real problem to me, JUnit's `@{Before,After}Class` methods are static (unlike TestNG's), which means that I have to initialize/destroy a FileSystem after _each test_; that is expensive.

I'm not sure about how to tackle this problem, really. I _do_ want to avoid having to create a `FileSystem` for each and every test, especially for classes which do not need to have one in the first place; and ideally I'd like to have the filesystem created and closed per test _class_ not per test, and have the test class initialize it with the contents needed, not the base class.

This is a piece of cake with TestNG, not so with JUnit :(